File: Python/1012.py
line1=input().split(" ")
a, b, c = line1
print("TRIANGULO: {:.3f}".format((float(a)*float(c))/2))
print("CIRCULO: {:.3f}".format((float(c)*float(c))*3.14159))
print("TRAPEZIO: {:.3f}".format(((float(a)+float(b))*float(c))/2))
print("QUADRADO: {:.3f}".format(float(b)*float(b)))
print("RETANGULO: {:.3f}".format(float(a)*float(b)))

File: uriOnlineJudge/pascal/1012.pas
var
    a, b, c : double;
begin
    readln(a, b, c);
    writeln('TRIANGULO: ', ((a*c)/2):0:3);
    writeln('CIRCULO: ', (3.14159*(c*c)):0:3);
    writeln('TRAPEZIO: ', (((a+b)*c)/2):0:3);
    writeln('QUADRADO: ', (b*b):0:3);
    writeln('RETANGULO: ', (a*b):0:3);
end.
